codeclimate-radon

```diff
@@ -2,16 +2,26 @@
 
 import json
 import os.path
+import sys
 
 threshold = "b"
+binstub = "radon3"
 include_paths = ["."]
 
 if os.path.exists("/config.json"):
     contents = open("/config.json").read()
     config = json.loads(contents)
 
-    if config.get("config") and config["config"].get("threshold"):
-        threshold = config["config"]["threshold"].lower()
+    if config.get("config"):
+        if config["config"].get("python_version"):
+            version = config["config"].get("python_version")
+            if version[0] == "2":
```

Comment:
I don't think that's an unreasonable point. Since this is only discoverable via the documentation anyway, we have a chance to be very clear about what valid options are. I'm sold.